app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -1,7 +1,7 @@
 .l-header {
-  background: $color-green-primary;
-  min-height: $baseline-unit*11;
-  position: relative;
+  width: 100%;
+  z-index: 200;
+  min-height: 55px;
```

Comment:
Not sure I follow, what was wrong with `min-height: $baseline-unit*11;` that was there before (or just using a higher multiple than 11)?